packages/map-template/src/components/Map/Map.jsx

```diff
@@ -98,19 +102,42 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
         }
     }, [filteredLocations, filteredLocationsByExternalIDs, mapsIndoorsInstance]);
 
+    /*
+     * React to changes in bearing and pitch props and set them on the map if mapsIndoorsInstance exists.
+     */
+    useEffect(() => {
+        if (mapsIndoorsInstance) {
+            if (bearing) {
+                mapsIndoorsInstance.getMapView().rotate(parseInt(bearing));
+            }
+            if (pitch) {
+                mapsIndoorsInstance.getMapView().tilt(parseInt(pitch));
+            }
+        }
+    }, [bearing, pitch, mapsIndoorsInstance]);
+
```

Comment:
```suggestion
    useEffect(() => {
        if (mapsIndoorsInstance) {
            if (pitch) {
                mapsIndoorsInstance.getMapView().tilt(parseInt(pitch));
            }
            if (bearing) {
                mapsIndoorsInstance.getMapView().rotate(parseInt(bearing));
            }
        }
    }, [bearing, pitch, mapsIndoorsInstance]);

```

Reply:
Changed the order so they are always consistent, first the check for pitch and then for bearing.